### PR TITLE
File 302

### PR DESCRIPTION
**Pull request by YuanZHANG910**
File 302: no ready for merge as there are no unit tests yet, the pull request is for comparison.

### CODE REVIEW

**Comment by YuanZHANG910** [comment]
File-302: sorry for take this so long, I had some difficulty with testing. 
The tests I have done are IT tests covers 400, 403, 404, 413, 423 and 500, now ready for review and merge. Thanks

File: app/uk/gov/hmrc/fileupload/frontendGlobal.scala [hunk 1]
```diff
@@ -32,18 +33,35 @@ import uk.gov.hmrc.fileupload.notifier.{NotifierRepository, NotifierService}
...
 
 import scala.concurrent.Future
 
-object FrontendGlobal
-  extends DefaultFrontendGlobal {
+object FrontendGlobal extends GlobalSettings with FrontendFilters with GraphiteConfig
+  with RemovingOfTrailingSlashes with Routing.BlockingOfPaths with ErrorAuditingSettings with ShowErrorAsJson {
+
+  lazy val appName = Play.current.configuration.getString("appName").getOrElse("APP NAME NOT SET")
+  lazy val enableSecurityHeaderFilter = Play.current.configuration.getBoolean("security.headers.filter.enabled").getOrElse(true)
```
Comment (konradwudkowski) [inline app/uk/gov/hmrc/fileupload/frontendGlobal.scala]: Are all of these sucurity filters a conscious decision to be included as part of enabling json responses?

File: it/uk/gov/hmrc/fileupload/ShowErrorAsJsonISpec.scala [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+      val contentType = result.header("Content-Type").get.split(";")
+
+      contentType(0) shouldBe MimeTypes.JSON
+    }
+
+
+  }
+
+  def uploadDummyFile(envelopeId: EnvelopeId, fileId: FileId): WSResponse = {
```
Comment (konradwudkowski) [inline it/uk/gov/hmrc/fileupload/ShowErrorAsJsonISpec.scala]: it's not very clear to me why calling this method would casue a 500 error

**Comment by konradwudkowski** [comment]
I looked at tests and I don't understand how they are testing if exceptions are properly handled.  You are not throwing any exceptions at all. Also just as an experiment I removed all your code changes in frontendGlobal and rerun your new tests and they still passed which means they may be not testing anything... Could you verify this please.

**Pull request closed without merging**